.neutrinorc.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
         entry: 'test-view/index.jsx',
         title: 'Treeherder Test View',
       },
+      pushhealthview: {
+        entry: 'push-health-view/index.jsx',
```

Comment:
What do you think about `s/push-health-view/push-health/` and `s/Push Health View/Push Health/` ? :-)

Reply:
I like it.  Done.